### PR TITLE
update README-ja.md

### PR DESCRIPTION
**Pull request by koh7**
https://github.com/jlevy/the-art-of-command-line/issues/566

### CODE REVIEW

File: README-ja.md [hunk 1]
```diff
@@ -118,13 +137,28 @@
 
 - `{`...`}`を使った中括弧展開によって、似たようなコマンドを複数回入力しなくて済む。例えば、 `mv foo.{txt,pdf} some-dir` (両方のファイルを移動させる), `cp somefile{,.bak}` (`cp somefile somefile.bak` と展開される)、`mkdir -p test-{a,b,c}/subtest-{1,2,3}` (すべての可能な組み合わせでディレクトリが作られる).
 
+- 展開の順序は括弧→チルダ、パラメータや変数、計算機号、コマンド置換(左から右)→文字の分割→ファイル名の順だ。(例えば、{1..20}のような範囲は{$a..$b}というようには表現できない。seqやforループを使ってこんな風に表すことができる。seq $a $b or for((i=a; i<=b; i++)); do ... ; done
```
Comment (doublemarket) [inline README-ja.md]: How about quoting those commands and the snippet with backquotes like `seq` or `seq $a $b or for((i=a; i<=b; i++)); do ... ; done`?

File: README-ja.md [hunk 2]
```diff
@@ -164,11 +198,21 @@
 
 - 特権レベルでコマンドを実行するとき、rootでの実行には`sudo`、他のユーザの場合は`sudo -u`を利用。`su` または `sudo bash`で、シェルがそのユーザで起動する。`su -`でrootまたは他のユーザで新たにログインした状態がシミュレートされる。
 
+-異なるユーザーのシェルに移りたいと時は`su username`や`su - username`を使おう。"-"を用いることでまるで他のユーザーがログインしたかのように環境を得ることができる。スイッチ先のユーザーのパスワードを求められる。
```
Comment (doublemarket) [inline README-ja.md]: Please put a space between `-` and 異なる so that this can be a list item.

**Comment by koh7** [comment]
@doublemarket Can I on the authors' list?  If it's OK, please check the pull request below.  
If not, just close it. Thanks.
https://github.com/jlevy/the-art-of-command-line/pull/570

**Comment by doublemarket** [comment]
@koh7 Thanks for the quick response 👍 

@jlevy Hey, this PR is great to make 🇯🇵 version up-to-date to the English version. Please merge this. Additionally please merge #570 as well if it's valid. Thank you!

**Comment by jlevy** [comment]
👍 Great. Thank you @doublemarket @koh7 and appreciate the patience on merging!

**Pull request merged**